qir/qat/Rules/FactoryConfig.hpp

```diff
@@ -15,12 +15,12 @@ namespace quantum
         void setup(ConfigurationManager& config)
         {
             config.setSectionName("Transformation rules", "Rules used to transform instruction sequences in the QIR.");
-            config.addExperimentalParameter(
-                optimize_result_one_, true, false, "optimize-result-one",
+            config.addParameter(
```

Comment:
Yes, this could have been called out more clearly - will do that next time. By way of explanation, these are necessary changes for the tests to pass with the requirements set out from my last call with @bettinaheim .